src/test/java/org/irods/Operations/ZoneOperationsTest.java

```diff
@@ -68,17 +69,17 @@ public void testAddingRemovingAndModifyingZones() {
             );
 
             for (Map.Entry<String, String> entry : propertyMap) {
-                String property = entry.getKey();
+                ZoneProperty zoneProperty = ZoneProperty.valueOf(entry.getKey().toUpperCase());
                 String value = entry.getValue();
 
-                res = client.zoneOperations().modify(rodsToken, zoneName, property, value);
+                res = client.zoneOperations().modify(rodsToken, zoneName, zoneProperty, value);
                 logger.debug(res.getBody());
                 assertEquals("Modifying zone request failed", 200, res.getHttpStatusCode());
-                assertEquals("Modifying zone failed", 0,
+                assertEquals("Modifying zone with property " + zoneProperty.getValue() + " failed", 0,
                         getIrodsResponseStatusCode(res.getBody()));
 
                 // Capture the new name of the zone following its renaming.
-                if ("name".equals(property)) {
+                if ("name".equals(zoneProperty.getValue())) {
```

Comment:
Instead of comparing a string, you can compare the actual enum.
```java
if (ZoneProperty.NAME == zoneProperty) {
```

Reply:
Oops. We already pounded this.

Open an issue for this and we'll handle it later.

Reply:
Ahh makes sense, created #86